### PR TITLE
Fix the large popup for editing forum categories

### PR DESCRIPTION
**Pull request by motechFR**

### CODE REVIEW

**Review by mattcasey — CHANGES_REQUESTED** [review]
this looks like a bug introduced by @msadura in this PR: https://github.com/charmverse/app.charmverse.io/pull/1909/files#diff-e45bfc88466d6c960d04e86d6b9efe64492813c7309aa2ec681ee2838cb16621. Rather than add more customization on this component, I wonder why not just use MUI's component directly. This seems to just be adding a bunch of defaults, but they don't fit every case so we're making our jobs even harder. The bug also exists on the proposal category popups, for example

**Comment by motechFR** [comment]
> https://github.com/charmverse/app.charmverse.io/pull/1909/files#diff-e45bfc88466d6c960d04e86d6b9efe64492813c7309aa2ec681ee2838cb16621

I'm counting 7 locations where we use this component, so i think it would be good to continue using it.

I did some checking, and this issue pops up in the member directory menu where you update the options (see with / without) screenshots below.

Went for the solution of making the popper simpler, and passing the style props from the outside.

I created a common setting for CharmEditor components that we can pass down.

Replicated intent of @msadura PR by passing these props down to the PDF, Media and Image selectors


Screenshot 2023-03-25 at 00 23 10

Screenshot 2023-03-25 at 00 23 42

File: components/common/CharmEditor/components/common/selectorPopupSizeConfig.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import type { Theme } from '@mui/material';
+import type { SxProps } from '@mui/system';
+
+// This is consumed by popup widgets in charm editor such as pdf or file selector to provide sufficient width
+export const selectorPopupSizeConfig: SxProps<Theme> = {
```
Comment (msadura) [inline components/common/CharmEditor/components/common/selectorPopupSizeConfig.ts:5]: I would move this directly  to `[MediaSelectionPopup.tsx]` - later on (not in a scope of this issue)we should refactor PDF and Image selector components to use this one as well, and this style will only be needed there.

File: components/common/PopperPopup.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ interface PopperPopupProps {
   onClose?: () => void;
   onOpen?: () => void;
   onClick?: () => void;
+  sx?: SxProps<Theme>;
```
Comment (msadura) [inline components/common/PopperPopup.tsx:16]: tiny stuff - lets call it `paperSx` to explicitly tell where those styles will be applied

**Review by msadura — APPROVED** [review]

File: components/common/PopperPopup.tsx (hunk 2 above)
Comment (motechFR) [reply components/common/PopperPopup.tsx]: Sounds good, made the update

**Review by mattcasey — APPROVED** [review]

File: components/common/PdfSelector/PdfSelector.tsx [hunk 3]
```diff
@@ -5,6 +5,8 @@ import MultiTabs from 'components/common/MultiTabs';
 import PopperPopup from 'components/common/PopperPopup';
 import { uploadToS3 } from 'lib/aws/uploadToS3Browser';
 
+import { selectorPopupSizeConfig } from '../CharmEditor/components/common/selectorPopupSizeConfig';
```
[the remark this answers is not in the document]
Comment (mattcasey) [reply components/common/PdfSelector/PdfSelector.tsx:8]: looks like PDFSelector should be inside CharmeDitor actually :P

**Pull request merged**